optaplanner-core/src/test/java/org/optaplanner/core/api/score/stream/AbstractConstraintStreamTest.java

```diff
@@ -119,14 +115,29 @@ protected static AssertableMatch assertMatchWithScore(int score, Object... justi
 
     protected static class AssertableMatch {
 
-        private int score;
-        private List<Object> justificationList;
+        private final int score;
+        private final List<Object> justificationList;
 
         public AssertableMatch(int score, Object... justifications) {
             this.justificationList = Arrays.asList(justifications);
             this.score = score;
         }
 
+        public boolean isEqualTo(ConstraintMatch constraintMatch) {
+            if (score != ((SimpleScore) constraintMatch.getScore()).getScore()) {
+                return false;
+            }
+            List<Object> actualJustificationList = constraintMatch.getJustificationList();
+            if (actualJustificationList.size() != justificationList.size()) {
+                return false;
+            }
+            /*
+             * We can not simply compare the lists, since the elements may be in different orders. The order of these
+             * lists is not relevant.
```

Comment:
Code style nitpick: Would you mind if we turn this into a 2 line comment line this:
// We can not ...
// lists is not relevant.
 Or even
// We can not ...
/// The order of ...

4 lines lost for a 2 line comment breaks the visual structure for me a bit.